pkgs/tools/filesystems/rmount/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchgit, makeWrapper, lib, pkgs ? import <nixpkgs> {} }:
+with pkgs;
```

Comment:
Derivations in `nixpkgs` *never* use `pkgs`, and neither would `import <nixpkgs>`. `mkDerivation` and `callPackage` handles dependency injection.

Additionally, using `with` is an issue, as here it makes everything available without proper dependency injection.